change/@fluentui-codemods-f663365a-ad02-46e0-9db5-a5ad17364c24.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
It looks like you accidentally added an empty file to codemods? Can you remove that file and this change file?

Reply:
hmm, ok, but weird. i thought it was a whitespace issue. i'll check again.